central/notifiers/common_test.go

```diff
@@ -198,7 +199,7 @@ func TestGetAnnotationValueReturnsDefaultIfNoStoreReturnsError(t *testing.T) {
 
 	alert := fixtures.GetAlert()
 
-	nsStore.EXPECT().SearchNamespaces(gomock.Any(), gomock.Any()).Return(nil, errors.New("testttt"))
+	nsStore.EXPECT().SearchNamespaces(gomock.Any(), gomock.Any()).Return(nil, errors.New(fixtureconsts.Cluster1))
```

Comment:
Is the error text used in a UUID field in the flow being tested ?

Reply:
this is just a mock in here.  I really didn't even need to change the line.  I do test some of that error testing of the UUID in PR 4/4